nucliadb/nucliadb/reader/api/v1/knowledgebox.py

```diff
@@ -50,14 +45,13 @@
 @requires(NucliaDBRoles.MANAGER)
 @version(1)
 async def get_kbs(request: Request, prefix: str = "") -> KnowledgeBoxList:
-    ingest = get_ingest()
+    driver = await get_driver()
+    txn = await driver.begin()
     response = KnowledgeBoxList()
-    async for kb_id in ingest.ListKnowledgeBox(KnowledgeBoxPrefix(prefix=prefix)):  # type: ignore
-        if kb_id.slug == "":
-            slug = None
-        else:
-            slug = kb_id.slug
-        response.kbs.append(KnowledgeBoxObjSummary(slug=slug, uuid=kb_id.uuid))
+    async for slug in KnowledgeBox.get_kbs(txn, prefix):
+        uuid = await KnowledgeBox.get_kb_uuid(txn, slug)
```

Comment:
Is there a way to get slug and uuid at the same time from maindb to avoid having this nested lookup?

Reply:
I don't think so. We'd need to do two lookups in any case I can think...

Reply:
getKbs needs to answer the kbid also

Reply:
Done

Reply:
Maybe it's not part of the initial scope of the PR anyway, but we could use the scan function to iterate key, value pairs (slug and uuid in our case), just like it is done in the docs: https://github.com/tikv/client-py/

Reply:
I like it! But we should implement this for every transaction type (Redis and local). Do you want to do it in this PR or open a ticket?

Reply:
for me it's out of the scope. We are good for now and if we ever see this is a problem, we can fix it by using scan